src/utils/axios.ts

```diff
@@ -13,12 +13,26 @@ const isBackendError = (
   return !!(err as AxiosError<ErrorDto>).response?.data.message
 }
 
+const isV2Error = (
+  err: AxiosError<unknown>
+): err is AxiosError<IsomerErrorDto> => {
+  return !!(err as AxiosError<IsomerErrorDto>).response?.data.error
```

Comment:
actually if we are making the checks in both methods to ensure response & message are present, do we need this? We can safely do `return `${error.response?.data.error?.code}: ${error.response?.data.error?.message}`` right?

Reply:
hmm the checks are to avoid the contortion where we keep having to give a default even when we assert a shape: 

```
      error.response?.data.error?.code
        ? `${error.response?.data.error?.code}: `
        : ""
    }${error.response?.data.error?.message || defaultErrorMessage}`
```

Reply:
i have removed the checks & default value as for IsomerExternalErrors it should be a established contract that presence of code & message is guaranteed